PushNotifications.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do we need this file? Can we delete it and things still be okay?

Reply:
This file is autogenerated. We can delete it, but Xcode will keep adding it to the repository. We can ignore it, but technically it is's one of the files that define the workspace.